python/lsst/ip/isr/isrTaskLSST.py

```diff
@@ -0,0 +1,1219 @@
+        dtype=bool,
+        doc="Widen bleed trails based on their width.",
+        default=True,
+    )
+    doCameraSpecificMasking = pexConfig.Field(
+        dtype=bool,
+        doc="Mask camera-specific bad regions?",
+        default=False,
+    )
+    masking = pexConfig.ConfigurableField(
+        target=MaskingTask,
+        doc="Masking task."
+    )
+
+    # Interpolation options.
+    doInterpolate = pexConfig.Field(
+        dtype=bool,
+        doc="Interpolate masked pixels?",
+        default=True,
+    )
+    maskListToInterpolate = pexConfig.ListField(
+        dtype=str,
+        doc="List of mask planes that should be interpolated.",
+        default=['SAT', 'BAD'],
+    )
+    doSaveInterpPixels = pexConfig.Field(
+        dtype=bool,
+        doc="Save a copy of the pre-interpolated pixel values?",
+        default=False,
+    )
+
+    # Initial masking options.
+    doSetBadRegions = pexConfig.Field(
+        dtype=bool,
+        doc="Should we set the level of all BAD patches of the chip to the chip's average value?",
+        default=True,
+    )
+
+    # Brighter-Fatter correction.
+    doBrighterFatter = pexConfig.Field(
+        dtype=bool,
+        doc="Apply the brighter-fatter correction?",
+        default=False,
+    )
+    brighterFatterLevel = pexConfig.ChoiceField(
+        dtype=str,
+        doc="The level at which to correct for brighter-fatter.",
+        allowed={
+            "AMP": "Every amplifier treated separately.",
+            "DETECTOR": "One kernel per detector.",
+        },
+        default="DETECTOR",
+    )
+    brighterFatterMaxIter = pexConfig.Field(
+        dtype=int,
+        doc="Maximum number of iterations for the brighter-fatter correction.",
+        default=10,
+    )
+    brighterFatterThreshold = pexConfig.Field(
+        dtype=float,
+        doc="Threshold used to stop iterating the brighter-fatter correction.  It is the "
+        "absolute value of the difference between the current corrected image and the one "
+        "from the previous iteration summed over all the pixels.",
+        default=1000,
+    )
+    brighterFatterApplyGain = pexConfig.Field(
+        dtype=bool,
+        doc="Should the gain be applied when applying the brighter-fatter correction?",
+        default=True,
+    )
+    brighterFatterMaskListToInterpolate = pexConfig.ListField(
+        dtype=str,
+        doc="List of mask planes that should be interpolated over when applying the brighter-fatter "
+        "correction.",
+        default=["SAT", "BAD", "NO_DATA", "UNMASKEDNAN"],
+    )
+    brighterFatterMaskGrowSize = pexConfig.Field(
+        dtype=int,
+        doc="Number of pixels to grow the masks listed in config.brighterFatterMaskListToInterpolate "
+        "when brighter-fatter correction is applied.",
+        default=0,
+    )
+    fwhm = pexConfig.Field(
+        dtype=float,
+        doc="FWHM of PSF in arcseconds.",
+        default=1.0,
```

Comment:
What is this used for?

Reply:
It is used in the interpolation of masked pixels in the brighter-fatter correction. This is also from the original task, there are probably historical reasons to use this.

Reply:
I think we can/should change the name then to something that specifies that it's part of the BFE correction.

Reply:
This should have a name consistent with the others, like `brighterFatterFwhm` or `brighterFatterFwhmForInterpolation`.

Reply:
much better - thanks!